File: RoutineOrganizerDomain/Models/Agenda.cs
using System;
using System.Collections.Generic;


namespace RoutineOrganizerDomain.Models
{
    public class Agenda
    {
        
        public Guid Id {get;set;}
        public string Name { get; set; }
        public string Notes { get; set; }
        public DateTime Period { get; set;}

        public string Activities {get;set;} 
        public string Projects {get;set;} 
        public string Priorities {get;set;} 
        public string Prouds {get;set;} 



      //  public virtual ICollection<ActivityDay> Activities {get;set;} 
      //  public  virtual ICollection<Project> Projects {get;set;} 
      //  public  virtual ICollection<Priority> Priorities  {get;set;} 
      //  public  virtual ICollection<Proud> Prouds {get;set;} 

    }
}

File: client/src/app/_models/Agenda.ts
import { Activity } from './Activity';
import { Priority } from './Priority';
import { Project } from './Project';
import { Proud } from './Proud';

export interface Agenda {
    id: string;
    name: string;
    notes: string;
    period: string;
    activities?: string; // Activity[];
    priorities: string; // Priority[];
    projects?: string; // Project[];
    prouds?: string; // Proud[];

}
